profiles/features/musl/package.mask

```diff
@@ -1,6 +1,10 @@
 # Copyright 1999-2024 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
+# Marco Genasci <fedeliallalinea@gmail.com> (2024-02-28)
+# Binary package, which is compiled against glibc
+app-emulation/genymotion-bin
```

Comment:
Down below there is

```
# Binary package linked to glibc (rolling mask, use for all such binpkgs)
```
which people really should use more often. There are so many packages recently added that aren't using the rolling mask. :(